tests/Integration/Mail/SendingMarkdownMailTest.php

```diff
@@ -52,23 +52,27 @@ public function testEmbed()
         $email = app('mailer')->getSymfonyTransport()->messages()[0]->getOriginalMessage()->toString();
 
         $cid = explode(' cid:', (new Stringable($email))->explode("\r\n")
-            ->filter(fn ($line) => str_contains($line, 'Embed content: cid:'))
+            ->filter(fn ($line) => str_contains($line, ' content: cid:'))
+            ->first())[1];
+
+        $filename = explode('Embed file: ', (new Stringable($email))->explode("\r\n")
+            ->filter(fn ($line) => str_contains($line, ' file:'))
             ->first())[1];
 
         $this->assertStringContainsString(<<<EOT
-        Content-Type: application/x-php; name=$cid\r
+        Content-Type: application/x-php; name=$filename\r
         Content-Transfer-Encoding: base64\r
-        Content-Disposition: inline; name=$cid; filename=$cid\r
+        Content-Disposition: inline; name=$filename;\r
+         filename=$filename\r
+        Content-ID: <$cid>\r
         EOT, $email);
     }
 
     public function testEmbedData()
     {
         Mail::to('test@mail.com')->send($mailable = new EmbedDataMailable());
 
-        $mailable->assertSeeInHtml('Embed data content: cid:foo.jpg');
```

Comment:
Why it’s removed?

Reply:
`cid` is now automatically generated by DataPart as a random value, so it's been removed. Alternatively, should we just check for the existence of `cid` instead?

```php
$mailable->assertSeeInHtml('Embed data content: cid:');
```

Any better suggestions?

Reply:
From my point of view, yes we should check `cid` existence at least as you mentioned instead of removing the check totally.

Thanks